cernopendata/modules/fixtures/data/records/opera-author-list-multiplicity.json

```diff
@@ -525,7 +525,7 @@
       "Author-Lists"
     ],
     "date_created": "2010-2012",
-    "date_published": "2018",
+    "date_published": "2017",
```

Comment:
I get the reasoning behind this, but we _are_ publishing in 2018. Distinguishing via names (multiplicity, tau) rather than dates makes more sense (to me).

Reply:
OK.  We could change the PDF filenames... too bad that 2017 and 2018 will appear in the PDF content...

Reply:
I can change the dates in the PDF, if you want. I can add "multiplicity studies" and "tau neutrino appearance studies" respectively as a subtitle.